src/app/cli/src/init/client.ml

```diff
@@ -989,6 +992,46 @@ let pooled_user_commands =
          in
          print_string (Yojson.Safe.to_string json_response) ))
 
+let pending_snark_work =
+  let open Command.Param in
+  Command.async
+    ~summary:
+      "List of snark works in JSON format that are not available in the pool \
+       yet"
+    (Cli_lib.Background_daemon.graphql_init (return ()) ~f:(fun port () ->
+         Deferred.map
+           (Graphql_client.query_exn
+              (Graphql_queries.Pending_snark_work.make ())
+              port)
+           ~f:(fun response ->
+             let lst =
+               [%to_yojson: Cli_lib.Graphql_types.Pending_snark_work.t]
```

Comment:
Nit: Feel like there is too much nesting of functions in this call. I think it would be better to change `Graphql_types.Pending_snark_work.t` to `type t = Work.t array array`.